.github/FUNDING.yml

```diff
@@ -0,0 +1,3 @@
+# These are supported funding model platforms
```

Comment:
This file is postgres-specific (the "Sponsor" button at the top of the page). We should remove it.

There are also other postgres-specific contents, we should clean them up.